### PR TITLE
Minor Training Variable Consistency fix

### PR DESCRIPTION
**Pull request by CatalinVoss**
Half question & half PR, but if I follow the code correctly, these `outputs['outputs']` are already softmax'ed, so they're probs, not logits, yes? This tripped me up as I sat here double-softmaxing and washing out my probabilities for a little while.

### CODE REVIEW

**Comment by reuben** [comment]
Correct, thanks for submitting a PR. Could you also fix `create_inference_graph`?

**Comment by reuben** [comment]
I triggered tests in #3402
